### ISSUE TITLE
Waspy run take 12 seconds to kill or reload

### ISSUE THREAD

**Issue by autoferrit** [wasp/waspy#8]
Talking to @nhumrich in irc:

![image](https://user-images.githubusercontent.com/89895/29737212-f8a46e54-89be-11e7-8b76-eb13596b34cc.png)

Environment:
* Ubuntu 17.04
* Python: 3.6.2

**Comment by nhumrich** [comment]
This only effects the `httptoolstransport`. The easiest fix is to set `shutdown_wait_period`, and `shutdown_grace_period` to 0. Shutdown will then take 0 seconds.

shutdown_wait_period is the time to wait after getting a shutdown signal before actually shutting down.
shutdown_grace_period is the time to wait for connections to be empty before shutting down. 
(both in seconds)

I need to work on a better fix for this, but its looking a little bigger than I hopped. So those should help for now.
An even better fix, is that setting the app to `debug` should eliminate all graceful shutdowns. Also, the defaults for these values should probably prefer local dev, and allow prod to set them higher. 



The history for this is that in kubernetes, there is a race condition where your service gets a shutdown before its taken out of the load balancer. These setting allow you to tweek things, so that your service gets shutdown after connections actually stop, thus preventing service disruption during a rolling deploy. However, I have found that this can be solved in other ways, so I need to experiment removing these values now.
